modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -112,14 +112,21 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<ScrollInfo | nul
   const iframe = editor.iframeElement;
   const iframeStyle = iframe.style;
 
+  const handleClasses = (handler: (elm: string | Node | Node[], cls: string) => void) => {
+    handler(body, 'tox-fullscreen');
+    handler(documentElement, 'tox-fullscreen');
+    handler(editorContainer, 'tox-fullscreen');
+    if (SugarShadowDom.isOpenShadowHost(fullscreenRoot)) {
+      handler(fullscreenRoot.dom, 'tox-fullscreen tox-shadowhost');
```

Comment:
This while technically works, it only happens by chance. This should be 2 separate calls, one for each class.

Reply:
Fixed